src/components/plots/stackedAreaPlot.ts

```diff
@@ -124,7 +122,7 @@ export module Plots {
     }
 
     protected _extentsForProperty(attr: string) {
-      var primaryAttr = this._isVertical ? "y" : "x";
+      var primaryAttr = "y";
       if (attr === primaryAttr) {
```

Comment:
But this reads better. It makes the `if` statement more explicit (easier to read)